migrations/0058_order_history.up.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS order_history (
+    id bigserial PRIMARY KEY,
```

Comment:
Here and in other tables, switched to `bigserial` to avoid issues with exhausting `serial` in the future. Data can be `DELETE`-ed, transactions – rolled back, so the likelihood of exceeding `serial` is high.

With `bigserial` we've got up to `9223372036854775807`, which should be enough.